Cargo.lock

```diff
@@ -69,9 +69,9 @@ dependencies = [
 
 [[package]]
 name = "anstyle-wincon"
-version = "3.0.2"
+version = "3.0.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "1cd54b81ec8d6180e24654d0b371ad22fc3dd083b6ff8ba325b72e00c87660a7"
+checksum = "61a38449feb7068f52bb06c12759005cf459ee52bb4adc1d5a7c4322d716fb19"
```

Comment:
**Issue**: The checksum for anstyle-wincon has been updated. It is essential to verify that the new checksum corresponds accurately to the new version to prevent any security risks associated with corrupted or tampered packages. <br> **Fix**: Verify the new checksum '61a38449feb7068f52bb06c12759005cf459ee52bb4adc1d5a7c4322d716fb19' directly from the source or through a secure, trusted channel to ensure it is correct and has not been compromised. <br> **Code Suggestion**: 
 ```
 -checksum = "1cd54b81ec8d6180e24654d0b371ad22fc3dd083b6ff8ba325b72e00c87660a7"
 +checksum = "61a38449feb7068f52bb06c12759005cf459ee52bb4adc1d5a7c4322d716fb19"
 ```